pennylane/qcut/cutcircuit.py

```diff
@@ -52,9 +52,7 @@ def processing_fn(res):
 
     # Expand the tapes for handling Hamiltonian with two or more terms
     tape_meas_ops = tape.measurements
-    if tape_meas_ops and isinstance(
-        tape_meas_ops[0].obs, (qml.ops.Hamiltonian, qml.ops.LinearCombination)
-    ):
+    if tape_meas_ops and isinstance(tape_meas_ops[0].obs, qml.ops.LinearCombination):
```

Comment:
```suggestion
    if tape_meas_ops and isinstance(tape_meas_ops[0].obs, qml.ops.Sum
```
I assume we would have this same issue for sums too.

Reply:
Would be nice to have a `num_terms` property for such a case.